### PR TITLE
Teams [Profile]

### PR DESCRIPTION
**Pull request by michaelfurutin**
image

image

image

image

### CODE REVIEW

File: web-app-router/scenes/Portal/Profile/Teams/page/List/index.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          >
+            Role
+          </Typography>
+
+          <div className="py-3 border-b border-grey-100" />
+        </div>
+
+        <div className="contents">
+          <div className="flex items-center gap-x-4 px-2 py-4 border-b border-grey-100">
+            <Image className="w-12 h-12" alt="" src="" />
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Profile/Teams/page/List/index.tsx:51]: @michaelfurutin Let's handle case when image can't load. [Example](https://github.com/worldcoin/developer-portal/blob/dd70a6ea37265f7193f4fde2afe8092715524698/web-app-router/scenes/Onboarding/Join/page/Logo/index.tsx#L9)

File: web-app-router/scenes/Portal/Profile/Teams/page/TransferTeamDialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      <DialogPanel className="w-[36.25rem] grid gap-y-8">
+        <CircleIconContainer variant="info">
+          <ExchangeIcon />
+        </CircleIconContainer>
+
+        <Typography as="h3" variant={TYPOGRAPHY.H6}>
+          Transfer ownership
+        </Typography>
+
+        <p className="leading-6 font-500 text-16 text-center text-grey-500">
```
Comment (igorosip0v) [inline web-app-router/scenes/Portal/Profile/Teams/page/TransferTeamDialog/index.tsx:26]: @michaelfurutin Transfer modal is not finished, it requires a form and copy update.

**Review by igorosip0v — APPROVED** [review]

File: web-app-router/scenes/Portal/Profile/Teams/page/List/index.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            variant={TYPOGRAPHY.R4}
+            className="flex items-center px-2 py-4 leading-5 text-14 text-grey-500 border-b border-grey-100"
+          >
+            Owner
+          </Typography>
+
+          <div className="flex items-center px-2 py-4 border-b border-grey-100">
+            <Dropdown>
+              <DropdownButton>
+                <MoreVerticalIcon />
```
Comment (andy-t-wang) [inline web-app-router/scenes/Portal/Profile/Teams/page/List/index.tsx:66]: this is the Elements Icon

File: web-app-router/components/Icons/MoreVerticalIcon.tsx
Comment (andy-t-wang) [inline web-app-router/components/Icons/MoreVerticalIcon.tsx:1]: Actually I see this is the preferred name I'll go and change it when I change the dropdown

**Review by andy-t-wang — APPROVED** [review]

**Pull request merged**